cfg/validate.go

```diff
@@ -177,6 +177,10 @@ func isValidMetricsConfig(m *MetricsConfig) error {
 	if m.StackdriverExportInterval != 0 && m.CloudMetricsExportIntervalSecs != 0 {
 		return fmt.Errorf("exactly one of stackdriver-export-interval and cloud-metrics-export-interval-secs must be specified")
 	}
+	const maxPortNumber = 65535
+	if m.PrometheusPort > maxPortNumber {
+		return fmt.Errorf("prometheus-port must not be higher than the maximum allowed port number: 65535 but received: %d instead", m.PrometheusPort)
```

Comment:
If we've turned this into a variable, might as well as use it here.

```suggestion
		return fmt.Errorf("prometheus-port must not be higher than the maximum allowed port number: %v but received: %d instead", m.PrometheusPort, maxPortNumber)
```

Reply:
Thanks!